src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -86,6 +87,32 @@ public JsonAdaptedPerson(Person source) {
         gitStats = source.getGitStats();
     }
 
+    /**
+     * Checks whether given Image is the default profile image
+     * @param image image to check
+     * @return true, if given image is same as default
+     */
```

Comment:
We standardized the Javadoc format so that there is one empty line between the description and `@param`.